android/KMEA/app/src/main/assets/android-host.js

```diff
@@ -219,22 +210,19 @@ function console_debug(s) {
 }
 
 function updateKMSelectionRange(start, end) {
-  var ta = document.getElementById('ta');
-  console_debug('updateKMSelectionRange('+start+','+end+'): ta.selectionStart='+ta.selectionStart+' '+
-    '['+ta._KeymanWebSelectionStart+'] ta.selectionEnd='+ta.selectionEnd+' '+ta._KeymanWebSelectionEnd);
+  var context = keyman.context;
+
+  // console_debug('updateKMSelectionRange('+start+','+end+'): context.selStart='+ta.selectionStart+' '+
+  //   '['+ta._KeymanWebSelectionStart+'] context.selEnd='+ta.selectionEnd+' '+ta._KeymanWebSelectionEnd);
 
-  var selDirection = 'forward';
```

Comment:
So we are losing the ability to detect selection direction? Is that going to potentially cause issues?